src/pages/ProxyVotes.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+
+import IntlMessage from "../components/IntlMessage";
+import { AppState } from "../reducers";
+import {
+  Member,
+  MemberLookupTable,
+  GENESIS_MEMBER
+} from "../reducers/membersNew";
+
+interface OwnProps {}
+interface StateProps {
+  members: Member[];
+}
+interface DispatchProps {}
+interface Props {
+  memberAndVotes: [[Member, number]];
```

Comment:
P2: Why isn't it just a `Map`? This is a weird, unintuitive data structure.

Reply:
for sorting